src/instruments/src/Common/gauges.tsx

```diff
@@ -171,3 +174,119 @@ export const VerticalSegment: FC<VerticalSegmentProps> = ({ x, y, height, rangeS
         />
     );
 };
+
+export function splitDecimals(value, type) {
+    if (type === 'oil') {
+        value = value * 0.01 * 25;
+    } else if (type === 'vib') {
+        value = value < 0 ? 0.0 : value;
+    }
+    const decimalSplit = value.toFixed(1).split('.', 2);
+    return (decimalSplit);
+}
+
+export function valueRadianAngleConverter(value, min, max, endAngle, startAngle) {
```

Comment:
**issue:** those functions do not have parameter or return types
**thought:** I don't think `splitDecimals` should be concerned with whether the result is for oil or vib. perhaps that logic can be extracted somewhere else ?